### DISCUSSION TITLE
Wails V2: Using/starting wails from inside a Windows application's event loop?

### DISCUSSION THREAD

**Discussion by Robert-M-Muench** [wailsapp/wails#1802] [Q&A]
I have a standard Windows app that has its event loop and blits some stuff via a transparent window to the screen.

Now I want to show a wails window for the application settings when the user hits a specific shortcut. That means I want to start up wails within the standard event loop, handle all user interaction, and shut down the window when the user is done.

Is that somehow possible? Or does Wails capture the whole application event loop, and I can't move outside of Wails any longer?

**Comment by leaanthony** [comment]
Not possible right now. The best you can do is exec an external config program.

**Comment by Robert-M-Muench** [comment]
Is something like an embeddable Wails app planned?

**Reply by leaanthony** [reply]
Not in the near future
